playbook/app/pb_kits/playbook/pb_button/_button_mixins.scss

```
@@ -12,24 +12,30 @@ $pb_button_h_padding: 34px;
 $pb_button_hover_darken: 4%;
 $pb_button_border_width: 0px;
 
+@mixin variant_overrides($bg: $primary_action, $color: $text_dk_default, $border: transparent) {
```

Could we maybe call this mixin `pb_button_variant` so it's clear throughout the file?

```suggestion
@mixin pb_button_variant($bg: $primary_action, $color: $text_dk_default, $border: transparent) {
```

---

@thestephenmarshall Good call.  Struggled to come up with an intuitive name for this one.

I'll switch it to `pb_button_variant`.